src/test/java/bio/terra/service/dataset/DatasetAzureIntegrationTest.java

```diff
@@ -454,19 +457,42 @@ public void datasetIngestFileHappyPath() throws Exception {
     TestUtils.verifyHttpAccess(snapshotParquetUrl, Map.of());
     verifySignedUrl(snapshotParquetUrl, steward(), "rl");
 
+    List<String> drsIds = new ArrayList<>();
     for (AccessInfoParquetModelTable table : snapshotParquetAccessInfo.getTables()) {
       if (tablesToCheck.contains(table.getName())) {
         String tableUrl = table.getUrl() + "?" + table.getSasToken();
         TestUtils.verifyHttpAccess(tableUrl, Map.of());
         verifySignedUrl(tableUrl, steward(), "rl");
+
+        // The vocabulary table has file data so test Drs on that one
+        // TODO: once we have an endpoint to expose parquet data, we should use that mechanism here
+        if (table.getName().equals("vocabulary")) {
+          List<Map<String, String>> records = ParquetUtils.readParquetRecords(tableUrl);
+          assertThat("2 rows are present", records.size(), equalTo(2));
+          // Extract the DRS Ids
+          drsIds.addAll(
+              records.stream()
+                  .map(r -> r.get("vocabulary_reference"))
+                  .collect(Collectors.toList()));
+        }
       }
     }
 
+    // Assert that 2 drs ids were loaded
+    assertThat("2 drs ids are present", drsIds.size(), equalTo(2));
+    // Ensure that all DRS can be parsed
+    List<String> drsObjectIds =
+        drsIds.stream()
+            .map(DrsIdService::fromUri)
+            .map(DrsId::toDrsObjectId)
+            .collect(Collectors.toList());
+
     String fileId = result.getLoadFileResults().get(0).getFileId();
     String filePath = result.getLoadFileResults().get(0).getTargetPath();
 
     // Do a Drs lookup
     String drsId = String.format("v1_%s_%s", snapshotId, fileId);
+    assertThat("Expected Drs object Id exists", drsObjectIds.contains(drsId), equalTo(true));
```

Comment:
`assertThat(x, y, equalTo(true))` is the same as `assertThat(x, y)`.
```suggestion
    assertThat("Expected Drs object Id exists", drsObjectIds.contains(drsId));
```

Reply:
ooh, right, I always forget about that one